client/my-sites/email/email-providers-comparison/in-depth/index.tsx

```diff
@@ -45,6 +46,8 @@ const EmailProvidersInDepthComparison = ( {
 
 	return (
 		<>
+			<QueryProductsList />
+
```

Comment:
I know we have done this even in the parent component (stacked comparison) but I am wondering what would be the general rule or best practice, to keep using this way of querying products, or useDispatch. 

MY FEELING is that we are not choosing the best practice (just my feeling and opinion). It would be nice to ask about this.

Reply:
To be honest I don't know what's the best practice for this. I'm also not entirely sure how we would use `useDispatch()` for this. I think that's a discussion you could start in our email P2.